data/mods/Aftershock/README.md

```diff
@@ -1,10 +1,11 @@
 # Design Document
-Aftershock is a set of parallel dimensions to Cataclysm Prime that have until this point existed in a relatively Blob free constellation of planes.  These planes have had additional decades to expand upon the advances that dimensional travel and nether exposure can bring even if it's been mostly hidden behind corporate/military shells.  These worlds are cresting between cyberpunk and post-cyberpunk.  Uplifted intelligent animals and solar colonization are relatively recent occurrences in the last few decades but for many in this world it is a dystopia even before the Cataclysm reaches here.  Mega/Hive cities cover large parts of human habitation where the poor live on the lower levels and rarely if ever see large amounts of vegetation that grows on the upper levels outside of hydroponics farms.  Outside of the largest cities much of the planet has been declared nature preserves where decaying and dying towns eke out an existence that is both richer and poorer than what is experienced by city dwellers.  
+* As it becomes a total conversion mod it will look very different from the base game of Cataclysm: Dark Days Ahead and may not interact kindly with other mods.  If you've been away for a while you may wonder where some parts of Aftershock have gone.  In the process of becoming a total conversion mod we are eliminating items that no longer fit the theme.  For example, all the Mad Max aesthetics are currently removed.  For them to come back would require someone to write up a location/faction that made them make sense in a futuristic extrasolar colony.  
```

Comment:
```suggestion
* As it becomes a total conversion mod, it will look very different from the base game of Cataclysm: Dark Days Ahead and may not interact kindly with other mods.  If you've been away for a while, you may wonder where some parts of Aftershock have gone.  In the process of becoming a total conversion mod, we are eliminating items that no longer fit the theme.  For example, all the Mad Max aesthetics have been removed.  For them to come back would require someone to write up a location/faction that made them make sense in a futuristic extrasolar colony.  
```

Reply:
Have been implies we are done, there is still plenty of stuff that needs auditing (like refrigerators)